src/lib/sidebar/generateSidebarItemsByPaths.ts

```
@@ -106,3 +112,21 @@ function findOrCreateGroup(
 
   return group
 }
+
+function applySidebarGroupTemplate(
+  sidebarItems: OASidebarItem[],
+  sidebarGroupTemplate: (path: string, depth: number) => string,
+  currentDepth: number = 1,
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)